<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViosGui</class>
 <widget class="QMainWindow" name="ViosGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViosGui</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWindow{
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
     }

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabwidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>750</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="face">
     <attribute name="title">
      <string>Face Recognition</string>
     </attribute>
     <widget class="QGroupBox" name="face_label">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>440</y>
        <width>301</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #CECECE, stop: 1 #FFFFFF);
 }</string>
      </property>
      <property name="title">
       <string>Face Label</string>
      </property>
      <widget class="QTextEdit" name="label_editor">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_button">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="label_grid"/>
      </widget>
      <widget class="QPushButton" name="add_button">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="selected_images">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>291</width>
        <height>381</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #CECECE, stop: 1 #FFFFFF);
 }
 </string>
      </property>
      <property name="title">
       <string>Detection</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <widget class="QGraphicsView" name="graphicsView_detection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>155</width>
         <height>155</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView_processed">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cropped Face</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Detected Face</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="show_margins">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Margins</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="h_indent">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="v_indent">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>220</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>H</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>220</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Indent</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_button">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
      <widget class="QPushButton" name="discard_button">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>300</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
      <widget class="QPushButton" name="train_button">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
      <widget class="QLabel" name="stat_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="browse">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>23</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="menu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>631</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #CECECE, stop: 1 #FFFFFF);
 }</string>
      </property>
      <property name="title">
       <string>menu</string>
      </property>
      <widget class="QPushButton" name="prev_skip_button">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="prev_button">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="frame_skip_textedit">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="fwd_button">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fwd_skip_button">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>591</width>
         <height>29</height>
        </rect>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QRadioButton" name="mode_image">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="mode_video">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mode_recognize">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Recognize</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="object">
     <attribute name="title">
      <string>Object Detection</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuViOS">
    <property name="title">
     <string>ViOS</string>
    </property>
   </widget>
   <addaction name="menuViOS"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
